lib/jekyll-docs.rb

```diff
@@ -20,17 +18,13 @@ def init_with_program(prog)
       end
 
       def process(opts)
-        Dir.mktmpdir do |dest_dir|
-          options = opts.merge({
-            "serving"     => true,
-            "watch"       => false,
-            "config"      => File.expand_path("../../site/_config.yml", __FILE__),
-            "source"      => File.expand_path("../../site", __FILE__),
-            "destination" => dest_dir
-          })
-          Jekyll::Commands::Build.process(options)
-          Jekyll::Commands::Serve.process(options)
-        end
+        options = opts.merge({
+          "serving"     => true,
+          "watch"       => false,
+          "destination" => File.expand_path("../../site", __FILE__),
```

Comment:
@bryanjhv Ok, we can give it a shot. Since the Build command is gone, I suppose we don't need it after all, as it's only looking for the destination. That said, it does look for a `_config.yml`, doesn't it?